Specs/DateTime/Japanese/TimeParser.json

```diff
@@ -573,19 +547,18 @@
   },
   {
     "Input": "朝の8時15分前です。",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "朝の8時15分前",
         "Type": "time",
         "Value": {
-          "Timex": "T07:45",
+          "Timex": "T08:15",
```

Comment:
Based on the reslution change, I believe this is similar to the previous comment. Let's make sure we support other ways of telling time, if relevant in Japanese.

Reply:
same as above, fixed now